### PR TITLE
Added display name in metadata response

### PR DESCRIPTION
**Pull request by AdilKhurshid**
Issue: https://github.com/equinor/vds-slice/issues/122

### CODE REVIEW

File: internal/vds/metadatahandle.hpp [hunk 1]
```diff
@@ -19,6 +19,7 @@ class MetadataHandle {
 
     BoundingBox bounding_box() const noexcept (true);
     std::string crs()          const noexcept (true);
+    std::string get_displayname() const noexcept (true);
```
Comment (ErlendHaa) [inline internal/vds/metadatahandle.hpp:22]: I know the above methods (`crs` and `bounding_box`) and `get_axis` below follow different conventions on using a `verb` so you fall in-between to chairs here. I'd opt for dropping get her tho, as the new method is conceptually closer to the ones above.

**Review by ErlendHaa — COMMENTED** [review]
Nice! Tests are failing. Have a look the build logs in GitHub Actions or run them locally (`go test -race ./...` and update the failing test :)

File: internal/vds/vds.cpp [hunk 2]
```diff
@@ -308,7 +308,8 @@ struct response metadata(
     nlohmann::json meta;
 
     meta["crs"] = metadata.crs();
-
+    meta["displayName"] = metadata.get_displayname();
```
Comment (ErlendHaa) [inline internal/vds/vds.cpp:311]: This is 100% my fault, but after looking at the OpenVDS docs I think we want [InputFileName](https://osdu.pages.opengroup.org/platform/domain-data-mgmt-services/seismic/open-vds/cppdoc/class/classOpenVDS_1_1KnownMetadata.html#_CPPv4N7OpenVDS13KnownMetadata30ImportInformationInputFileNameEv) instead.

Comment (AdilKhurshid) [reply internal/vds/vds.cpp]: Sure! I replaced the display name with the input file name and function

File: internal/vds/metadatahandle.hpp (hunk 1 above)
Comment (AdilKhurshid) [reply internal/vds/metadatahandle.hpp]: I changed the function name to `input_filename`. Let me know if it follows the convention we are using or just suggest me the appropriate name.

**Review by ErlendHaa — COMMENTED** [review]
Looks good, but you also need to update the Metadata struct in `vds.go` which serves as documentation in swagger

File: internal/vds/vds.go [hunk 3]
```diff
@@ -109,10 +110,10 @@ func GetAxis(direction string) (int, error) {
...
 		case "time":      return AxisTime,      nil
 		case "sample":    return AxisSample,    nil
-		default:
-			options := "i, j, k, inline, crossline or depth/time/sample"
-			msg := "invalid direction '%s', valid options are: %s"
-			return -1, fmt.Errorf(msg, direction, options)
+	default:
+		options := "i, j, k, inline, crossline or depth/time/sample"
+		msg := "invalid direction '%s', valid options are: %s"
+		return -1, fmt.Errorf(msg, direction, options)
```
Comment (ErlendHaa) [inline internal/vds/vds.go:116]: This indentation is technically correct. But the change is incomplete. The cases should also be un-indented:
```
switch ...
case ...
case ...
default:
```

It's greatly encourage to fix stuff like this when you find it. Even when it's unrelated to the PR. From experience, that is the only way it's ever getting fixed. Nobody is gonna bother to create issues and stand-alone PRs just with minor fixed like this. But fixing them as part of some other work, like you do here is fast and simple. BUT in order to not distract from what's really going on in commit, these types of fixes should be granted there own small commits like: "Fix incorrect indentation in switch-case" or "Fix whitespace errors". That reduces noice in the diffs of the other commits. 

So complete the indentation and extract that (together with the other indentation changes) in a separate commit. And continue fixing stuff like this when you find it 🎉

**Review by ErlendHaa — COMMENTED** [review]
Looks good. Fix up and squash the history and do a force-push to get a clean history

File: internal/vds/vds.go (hunk 3 above)
Comment (AdilKhurshid) [reply internal/vds/vds.go:116]: Thanks for letting me. I didn't add this intentionally. It's the IDE who reformatted the file indentation and I mistakenly push that in the PR. It's good that you mentioned it, I will take care of that next time and add additional commits whenever needed. :)

File: internal/vds/vds.go [hunk 4]
```diff
@@ -80,6 +78,9 @@ type Metadata struct {
 	// Coordinate reference system
 	Crs string `json:"crs" example:"PROJCS[\"ED50 / UTM zone 31N\",..."`
 
+	// The original input file name
+	InputFileName string `json:"inputFileName" example:"EQ21200DZC21B-KPSDM-RAW-FULL-0535-DEPTH.sgy"`
```
[the remark this answers is not in the document]
Comment (AdilKhurshid) [reply internal/vds/vds.go]: Done

File: internal/vds/vds.go (hunk 3 above)
Comment (ErlendHaa) [reply internal/vds/vds.go:116]: Tip: I _always_ use `git add -p`. Then I see exactly what I add to each commit 👍

**Review by ErlendHaa — APPROVED** [review]
LGTM, merge away 🎉

**Comment by AdilKhurshid** [comment]
> LGTM, merge away 🎉

Can you merge this PR? I don't have write access to this repository.

**Pull request merged**